Dolphin scripts/Entrance Randomizer/lib/graph_creation.py

```diff
@@ -109,32 +109,36 @@ def create_edges(transitions_map: Mapping[tuple[int, int], tuple[int, int]]):
         output_text += (
             f'<edge source="{TRANSITION_INFOS_DICT[pairing[0]].area_id}" '
             + f'target="{TRANSITION_INFOS_DICT[pairing[1]].area_id}" isDirect="false" '
-            + f'id="{counter}" ></edge>\n'
+            + f'id="{counter}"></edge>\n'
         )
         counter += 1
     for pairing in connections_one_way:
         output_text += (
             f'<edge source="{TRANSITION_INFOS_DICT[pairing[0]].area_id}" '
             + f'target="{TRANSITION_INFOS_DICT[pairing[1]].area_id}" isDirect="true" '
-            + f'id="{counter}" ></edge>\n'
+            + f'id="{counter}"></edge>\n'
         )
         counter += 1
     return output_text
 
 
 def create_graphml(
     transitions_map: Mapping[tuple[int, int], tuple[int, int]],
+    temp_disabled_exits: Sequence[tuple[int, int]],
     seed_string: SeedType,
     starting_area: int,
 ):
-    graphml_text = f"""\
-<?xml version="1.0" encoding="UTF-8"?>
-<graphml>
-  <graph id="Graph" uidGraph="1" uidEdge="1">
-    {create_vertices(transitions_map, starting_area)}
-    {create_edges(transitions_map)}
-  </graph>
-</graphml>"""
+    all_transitions: Mapping[tuple[int, int], tuple[int, int]] = transitions_map.copy()
```

Comment:
By using copy, this is locked down to being a dict, you can simply update the expected type

```python
    transitions_map: dict[tuple[int, int], tuple[int, int]],
    temp_disabled_exits: Sequence[tuple[int, int]],
    seed_string: SeedType,
    starting_area: int,
):
    all_transitions = transitions_map.copy()
```

Reply:
Upper change applied.